src/index.html

```diff
@@ -1,19 +1,2821 @@
 <!doctype html>
-<html lang="en">
+<html
```

Comment:
Code seems correct. Please add DEMO link in the PR as instructions requirements

Reply:
Eu fiz a tradução em português mas tem as duas, EN e PT, vou adicionar o link. Obrigada

Reply:
Não consegui corrigir o carregamento em Inglês como primeira opção, eu queria que aparecesse primeiro em inglês e que ao mudar fosse para o português, não consegui fazer isso.

Reply:
[DEMO LINK](https://DevMichelleBarcelos.github.io/layout_KateVR/).

Reply:
Eu coloquei o DEMO LINK na descrição desde o primeiro commit, não entendi porque aparece como descrição não disponível